Utils.py

```diff
@@ -55,8 +55,7 @@ def snes_to_pc(value):
 
 
 def cache_argsless(function):
-    if function.__code__.co_argcount:
-        raise Exception("Can only cache 0 argument functions with this cache.")
+    assert function.__code__.co_argcount, "Can only cache 0 argument functions with this cache."
```

Comment:
should this really be in this PR? i don't really mind, but maybe unrelated cleanup commits should go in separate PRs  going forward if hey are not required by the change so it's easier to git blame (especially if the PR ends up being squashed)